source/tasks/OctoInstaller/index.ts

```diff
@@ -23,7 +23,7 @@ async function run(){
             return;
         }
 
-        console.log(`Failed to resolve latest octo version. Using embedded version. ${error}`);
+        console.log(`Failed to resolve octo version ${version}. Using embedded version. ${error}`);
```

Comment:
Can we include the version number of the embedded version?

Reply:
Not in that message, but I've added a message after we start resolving it:
```
Using embedded Octo command line tool (version ${option.version}).
```